### ISSUE TITLE
Issues?

### ISSUE THREAD

**Issue by Dboehm15** [Gabeiscool420/AutoGain-SoundSage#4]
I saw you post this repo looking for contributors but you didn't really say any topic you'd like help with. To be completely honest I don't know anything about audio processing, but I have been tinkering with integrating OpenAI models into different applications. I suggest create some issues for things you'd like help with or make a TODO.md list.

**Comment by Gabeiscool420** [comment]
hey there! That's actually perfect because I'm just an Audio Engineer and although I understand the fundamentals of audio processing I'm still beginning to learn programming, I know what needs to be done on the Audio side of things it's just the coding that I'm very new with. 

where I am at currently is I have this functioning program that automates gain-staging and i would like to integrate it with the LLM/OpenAI model, if you take a look at the README.md at the beginning of the SoundSage---LLM-Audio-Processing directory i have it sort of all planned out, maybe you can take a look at it and tell me what you think. 

in short the idea is that i would like to prompt an openAI chat interface to find specified files in my system and move them to the soundsage/workbench/preprocess folder before sending a signal to another script to begin the AutoGain interaction. I am 100% certain that I am missing some aspect of how it works but I'm determined to keep going even though i dont see any light haha!

thank you for responding to my post, i am very grateful for any sort of help or guidance you can give me

Gabriel

here is a TODO.md:

https://github.com/Gabeiscool420/SoundSage---LLM-Audio-Processing/blob/SoundSage---LLM-Integration/TODO.md
